2_natural_language_processing/app/README.md

```diff
@@ -0,0 +1,40 @@
+# Full Stack Sentiment Analysis Application
```

Comment:
Let's rename this directory from `app` to `ml_project_sentiment_analysis_application`.